src/data/set/basic.lean

```diff
@@ -441,6 +441,21 @@ lemma exists_mem_of_nonempty (α) : ∀ [nonempty α], ∃x:α, x ∈ (univ : se
 instance univ_decidable : decidable_pred (∈ @set.univ α) :=
 λ x, is_true trivial
 
+lemma ne_univ_iff_exists_not_mem {α : Type*} (s : set α) : s ≠ univ ↔ ∃ a, a ∉ s :=
+⟨λ h, by { by_contradiction w, exact h (eq_univ_iff_forall.mpr (not_exists_not.mp w)), },
+  λ h w, by simpa [w] using h⟩
+
+lemma not_subseteq_iff_exists_mem_not_mem {α : Type*} {s t : set α} :
```

Comment:
`subseteq` -> `subset` seems to be the naming convention.

The proof can be simplified to `by simp [subset_def]`.

Reply:
Much nicer!